routes/socket/game/start-game.js

```diff
@@ -525,6 +525,24 @@ const beginGame = game => {
 		},
 		process.env.NODE_ENV === 'development' ? 100 : experiencedMode ? 5400 : 9000
 	);
+
+	for (let affectedPlayerNumber = 0; affectedPlayerNumber < game.publicPlayersState.length; affectedPlayerNumber ++) {

```

Comment:
This is fine, but it could theoretically be swapped to a `forEach()` on every object in `game.publicPlayerState` and directly pass the username to line 533. 

(Not necessary, but just an idea if you wanted)